listings/urls.py

```diff
@@ -6,9 +6,9 @@
 from listings.views import ListingViewSet
 
 router = routers.DefaultRouter()
-router.register(r'listings', ListingViewSet, basename='listing')
+router.register(r'listings', ListingViewSet, basename='listings')
 
 
 urlpatterns = [
-    path('', include(router.urls))
+    path('', include((router.urls, 'listings'), namespace='listings'))
```

Comment:
What's the purpose of adding 'listings' to the `include()` call? The top level urls.py has
`path('listings/', include('listings.urls'))`, and the router has 'listings'... does this make the url `listings/listings/listings`?

Reply:
The 'listings' in the include call refers to the app name, because it's not allowed for me to specify "namespace" here, and not also supply the app name. The namespace parameter to the path call allows me to refer to this URL using "listings:" in my calls to django reverse later in my tests. You can see it in use in the tests I wrote in this PR.

Reply:
Oh, makes sense. I know what namespace is used for, didn't know you had to supply the app name to include().